services/graphql-server/src/graphql/resolvers/platform/content.js

```diff
@@ -659,7 +662,11 @@ module.exports = {
         maxResults,
         ...(pageToken && { pageToken }),
       };
-      return googleDataApiClient.request('youtube.playlistItems', payload);
+      const response = await googleDataApiClient.request('youtube.playlistList', { part: 'id', id: playlistId });
+      if (getAsArray(response, 'items').length > 0) {
+        return googleDataApiClient.request('youtube.playlistItems', payload);
```

Comment:
@Shinsina - is it necessary to use two separate requests for this?

In other words, do you have to first call `youtube.playlistList` to determine if there are any results?  Or, can you figure that out with the second call and either return the response or an empty array accordingly?

Reply:
@brandonbk This should now be improved in subsequent commits. I hadn't actually looked at how the responses were structured both responses return an "items" array and return the same data I was simply trying to reuse the logic found in existing GraphQL queries in order to do this validation, it should go to note however that as far as I can tell nothing is using these validation queries as the Leaders blocks call for the companies in blocks (I hadn't necessarily determined how the blocks were structured or the size of them but it's clear it calls for multiple at once) so the validation queries never have the chance to intervene.

<img width="1034" alt="Screen Shot 2021-08-16 at 9 23 49 AM" src="https://user-images.githubusercontent.com/46794001/129579153-f6b822e8-3728-4883-b347-5e2cf8dd8442.png">